lib/bin/gpa-games-cli

```diff
@@ -67,8 +67,7 @@ class HexCliGameState(UiGameState):
             raise("Unrecognized player, \"{}\"".format(ui_player))
 
     def reset_state(self):
-        self.state.reset(*self.state.board.size())
-        return self
+        self.state.reset()
```

Comment:
The return here was useful in case the user of this class wants to do method chaining. It's true that we're not using that now so it's probably fine, but in general I like to return `self` after mutation methods.